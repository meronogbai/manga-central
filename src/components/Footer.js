import styles from '../styles/Footer.module.css';

const Footer = () => (
  <footer className={styles.footer}>
    Made with
    {' '}
    <span role="img" aria-label="heart">❤️</span>
    {' '}
    by Meron Ogbai
  </footer>
);

export default Footer;
