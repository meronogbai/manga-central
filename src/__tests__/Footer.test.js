import { BrowserRouter } from 'react-router-dom';
import { create } from 'react-test-renderer';
import Footer from '../components/Footer';

test('renders correctly', () => {
  const Placeholder = () => (
    <BrowserRouter>
      <Footer />
    </BrowserRouter>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
