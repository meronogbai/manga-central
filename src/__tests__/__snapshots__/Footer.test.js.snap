// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="footer"
>
  Made with
   
  <span
    aria-label="heart"
    role="img"
  >
    ❤️
  </span>
   
  by Meron Ogbai
</footer>
`;
